### ISSUE TITLE
Related videos on YouTube are not being removed

### ISSUE THREAD

**Issue by wietop** [jprusik/custom-yt-embed#1]
I have used the converter for years and have never had problem. I use it for my website because I include YouTube videos occasionally and don't want the related videos showing at the end. Lately, I choose the code to remove them, but they still appear. I know YouTube is always trying to prevent customization and was wondering if that feature on the converted no longer works.

**Comment by jprusik** [comment]
~Recognizing you reported this quite some time ago, are you still seeing this issue? I've been unable to reproduce the issue with the default video, which correctly displays (or not) the related videos on the basis of that option selection.~

From [YouTube's API docs:](https://developers.google.com/youtube/player_parameters)

> Note: This parameter is [changing](https://developers.google.com/youtube/player_parameters#release_notes_08_23_2018) on or after September 25, 2018.
> 
> Prior to the change, this parameter indicates whether the player should show related videos when playback of the initial video ends.
> 
>     If the parameter's value is set to 1, which is the default value, then the player does show related videos.
>     If the parameter's value is set to 0, then the player does not show related videos.
> 
> After the change, you will not be able to disable related videos. Instead, if the rel parameter is set to 0, related videos will come from the same channel as the video that was just played.

😭

**Comment by wietop** [comment]
I had completely forgotten about this issue as I opted to just download the YouTube videos and upload them to my blog site. I just now tried to embed one using your website and the related videos showed at the end. Don't worry about this problem. I have found a workaround and it seems to be working fine. I'm sorry for any confusion.

Sent with [ProtonMail](https://protonmail.com/) secure email.

------- Original Message -------
